### ISSUE TITLE
Q2: train a model on a training set. 

### ISSUE THREAD

**Issue by andmikey** [andmikey/Thai-and-English-OCR#2]
(12 points) Write a script to train a model on a training set generated from the previous script. You can use any neural network structure you like, including extremely trivial ones. You can use a validation set however you like, or none at all. The restrictions are as follows:

* The model trained will have as input a single image in the format of the dataset and produce as output a character corresponding to the identifiers in the ThaiOCR dataset or corresponding Unicode values.
* The model must be trained in PyTorch and possible to run on mltgpu or mltgpu-2.  You may use HuggingFace tools if you like (not sure why you would, but just saying it in preparation for the next requirement).
* You may not use any pretrained models that can already recognize Thai or English characters to any degree.
* There must be at least one hidden layer and nonlinearity in the model.
* The script must save the trained model to a specified file.
* The script must allow for batching and for an arbitrary number of training epochs.
* Training must involve exactly one GPU when run on the MLT servers.
